### PR TITLE
Add Polish translations

### PR DESCRIPTION
**Pull request by HubertStefanski**
**When merged this pull request will:**
- Add missing polish translations
*Note:* Some translations were left out as most of the Polish community will be more familiar with their original names of certain things

### CODE REVIEW

**Comment by HubertStefanski** [comment]
I'll fix those build errors some time tomorrow, by the looks of it, it's just indentation on two files

**Comment by veteran29** [comment]
I'll review this when I'll be back from holidays.

File: addons/camera/stringtable.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
             <Czech>Určuje, zda by se měla výška kamery upravovat podle změn výšky terénu.</Czech>
             <Japanese>標高に合わせてカメラの高さを変化させます。</Japanese>
             <Korean>카메라의 높이가 지형의 고도 변화에 따라 변화되는지를 제어합니다.</Korean>
+            <Polish>Kontroluje czy kamera podążą względem wysokości terenu.</Polish>
```
Comment (veteran29) [inline addons/camera/stringtable.xml:34]:
```suggestion
            <Polish>Kontroluje czy kamera podąża względem wysokości terenu.</Polish>
```

File: addons/common/stringtable.xml [hunk 2]
```diff
@@ -569,15 +584,19 @@
...
             <English>Ascension Messages</English>
+            <Polish>Powiadomienia o wyznaczeniu Zeusa</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_AscensionMessages_Description">
             <English>Controls whether global popup messages are displayed when a player is assigned as Zeus.</English>
+            <Polish>Kontroluje czy po wyznaczeniu nowego gracza na Zeusa zostaje wyświetlona wiadomość globalna.</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_CameraBird">
             <English>Camera Eagle</English>
+            <Polish>Orla kamera</Polish>
```
Comment (veteran29) [inline addons/common/stringtable.xml:595]:
```suggestion
            <Polish>Orzeł przy kamerze</Polish>
```

File: addons/compositions/stringtable.xml [hunk 3]
```diff
@@ -3,33 +3,43 @@
...
             <English>CREATE CUSTOM COMPOSITION</English>
+            <Polish>STWÓRZ SWOJĄ KOMPOZYCJE</Polish>
         </Key>
         <Key ID="STR_ZEN_Compositions_EditCustomComposition">
             <English>EDIT CUSTOM COMPOSITION</English>
+            <Polish>EDYTUJ SWOJĄ KOMPOZYCJE</Polish>
         </Key>
         <Key ID="STR_ZEN_Compositions_CategoryCannotBeEmpty">
             <English>Category cannot be empty.</English>
+            <Polish>Pole kategorii nie może byc pusta.</Polish>
```
Comment (veteran29) [inline addons/compositions/stringtable.xml:18]:
```suggestion
            <Polish>Pole kategorii nie może być puste.</Polish>
```

File: addons/compositions/stringtable.xml [hunk 4]
```diff
@@ -3,33 +3,43 @@
...
         </Key>
         <Key ID="STR_ZEN_Compositions_NameCannotBeEmpty">
             <English>Name cannot be empty.</English>
+            <Polish>Pole nazwy nie może byc pusta</Polish>
```
Comment (veteran29) [inline addons/compositions/stringtable.xml:22]:
```suggestion
            <Polish>Pole nazwy nie może być puste.</Polish>
```

File: addons/compositions/stringtable.xml [hunk 5]
```diff
@@ -3,33 +3,43 @@
...
             <English>Select objects to create composition from</English>
+            <Polish>Wybierz obiekty do stworzenia kompozycji</Polish>
         </Key>
         <Key ID="STR_ZEN_Compositions_RandomizationOff">
             <English>Randomization disabled.</English>
+            <Polish>Wybór losowy wyłączony.</Polish>
         </Key>
         <Key ID="STR_ZEN_Compositions_RandomizationOn">
             <English>Randomization enabled.</English>
+            <Polish>Wybór losowy wyłączony.</Polish>
```
Comment (veteran29) [inline addons/compositions/stringtable.xml:42]:
```suggestion
            <Polish>Wybór losowy włączony.</Polish>
```

**Pull request merged**